perllib/FixMyStreet/Roles/ParallelAPI.pm

```diff
@@ -73,7 +73,7 @@ sub call_api {
     if (FixMyStreet->test_mode || $self->sample_data) {
         $start_call = 0;
         $data = $self->_parallel_api_calls(@_);
-    } elsif (-e $tmp) {
+    } elsif (-e $tmp && time-(stat($tmp))[9] < 3600) {
```

Comment:
Might be an idea to extract this magic `3600` somewhere more obvious & document it in case we need to change it later on.